RELEASING.md

```diff
@@ -27,6 +27,37 @@ You can run `make gorelease` that runs [gorelease](https://pkg.go.dev/golang.org
 
 You can check/report problems with `gorelease` [here](https://golang.org/issues/26420).
 
+## Verify changes for contrib repository
+
+If the changes in the main repository are going to affect the contrib repository, it is important to verify that the changes are compatible with the contrib repository.
+
+Follow the following steps in the local [contrib repository](https://github.com/open-telemetry/opentelemetry-go-contrib) to verify the changes.
+
+1. Pick the GIT SHA on the [main branch](https://github.com/open-telemetry/opentelemetry-go/commits/main) that you want to verify.
```

Comment:
These seem like steps relevant to contrib. Should they live in the contrib `RELEASING.md` instead?

Reply:
I can move this part to contrib and make a reference in this section.

Reply:
Blocked by https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5527